rskj-core/src/main/java/co/rsk/net/NodeBlockProcessor.java

```diff
@@ -152,15 +146,7 @@ public void processBlockHeaders(@Nonnull final MessageSender sender, @Nonnull fi
     }
 
     private boolean hasHeader(@Nonnull final BlockHeader h) {
-        if (hasBlock(h.getHash())) {
-            return true;
-        }
-
-        if (store.hasHeader(h.getHash())) {
-            return true;
-        }
-        
-        return false;
+        return hasBlock(h.getHash()) || store.hasHeader(h.getHash());
```

Comment:
We could invoke getHash only one, and save it in a variable

Reply:
Just changed the parameter to be the hash so we don't call it twice in this function. Calling it twice from the calling function was out of scope.